### DISCUSSION TITLE
Transfer Ownership

### DISCUSSION THREAD

**Discussion by nitro-codes** [community/community#23941] [New to GitHub]
<p>Hi,</p>
<p>I have a company here in Brazil called Nitrocodes and have been trying to find a way to contact the owner of the organization <a href="https://github.com/nitrocodes" rel="noopener nofollow ugc">https://github.com/nitrocodes</a> as it seems to be inactive since 2017. I don’t wanna create “iNitrocodes” or “TheNitrocodes” or variations if that account was never used since 2017, so I would like to ask if it would consider giving up the name or transfer the ownership.</p>
<p>Could you guys help me getting in contact with the owner? or something else?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>If there is no public contact information, we won’t be able to help you contact them, but if you think there’s a good chance it’s inactive, you can <a href="https://support.github.com/contact">open a ticket</a> and ask if you can have the username.</p>
<p>Just keep in mind that not all activity on GitHub is public so the answer could go either way!</p>

**Comment by 361709** [comment]
Transfer
